10-setup-htcondor.sh

```diff
@@ -7,6 +7,87 @@ if [ `id -u` = 0 ]; then
     exit 1
 fi
 
+
+#
+# Taken from creation/web_base/condor_startup.sh in the gwms 
```

Comment:
I guess we have to do this for now but we should try and get upstream to move this function to a sourceable `condor_lib.sh` file, like how they have `singularity_lib.sh`.